Firebase/InAppMessagingDisplay/FIRIAMDefaultDisplayImpl.m

```diff
@@ -69,6 +70,42 @@ + (NSBundle *)getViewResourceBundle {
   return resourceBundle;
 }
 
++ (void)displayCardViewWithMessageDefinition:(FIRInAppMessagingCardDisplay *)cardMessage
+                             displayDelegate:(id<FIRInAppMessagingDisplayDelegate>)displayDelegate {
+  NSBundle *resourceBundle = [self getViewResourceBundle];
+
+  if (resourceBundle == nil) {
+    NSError *error = [NSError errorWithDomain:kFirebaseInAppMessagingDisplayErrorDomain
+                                         code:FIAMDisplayRenderErrorTypeUnspecifiedError
+                                     userInfo:@{@"message" : @"resource bundle is missing"}];
```

Comment:
Nit: capitalize and add punctuation. Also, is `message` supposed to be localizedDescription?